src/prefect/cli/events.py

```diff
@@ -23,6 +27,18 @@ class StreamFormat(str, Enum):
     text = "text"
 
 
+def get_event_subscriber_type_for_context(
+    account: bool,
+) -> Type[PrefectEventSubscriber]:
+    api_url = PREFECT_API_URL.value()
+
+    if PREFECT_EXPERIMENTAL_EVENTS and (api_url is None or "/account" not in api_url):
```

Comment:
this `PREFECT_API_URL` check is a bit hacky, but because the tests use `localhost` urls for cloud urls, this is the best way I could readily think of. happy to do something more robust here if necessary

Reply:
doh I missed that - will use that here!

Reply:
updated in [0ec01b7](https://github.com/PrefectHQ/prefect/pull/13161/commits/0ec01b7733dfd056305e91883003ef58a088a296)